### ISSUE TITLE
EXP Farm

### ISSUE THREAD

**Issue by putaquerida** [finndev/PokeBuddy#1049]
Tokyo, Shinjuku Chuo Park : 35.6894568822069,139.69079732894897 140k per hour
Does anyone know a better location???

**Comment by ouji-san** [comment]
San juan Puerto Rico

**Comment by AthosAbreu** [comment]
what speed walking uses? and how much xp in san juan ?

**Comment by AthosAbreu** [comment]
putaquerida: what speed walking uses? and how much xp in shinjuku park

**Comment by ZzYaPoO** [comment]
@AthosAbreu  u still using PokeBuddy to bot? u dont scare u get ban from Game Master from Pokemon GO? u have any idea to prevent this?

**Comment by AthosAbreu** [comment]
I study one solution for this

**Comment by ZzYaPoO** [comment]
@AthosAbreu  what you study, we share together!

after my main account banned, i tried created new account for bot testing, speed 10 km/h, at Malaysia Penang area keep boting, never login via mobile device. just let it bot until 2X lvl, just start play back pokemon.

**Comment by putaquerida** [comment]
Use 30 Km/h, not teleport.

**Comment by AthosAbreu** [comment]
I saw many people talking about the same subject, ip ban , softban , actually, let's clarify,
1- The Niantic said nothing of Ban , or messages that speak out there 99% are false.
2- In the report site problems , ban , all answers are the same, ie they are not watching anything or doing anything about it .
3 emulator or Bot, they are perhaps considered cheating , but they use the GPS , and the source code of Google Maps , and if you stay in your region / city of origin , has no problem because it has no way to know , cause you can be bike , motorcycle , car, I do not know any things , that is , speed is relative.
4 - What did is set a time limit collection , exp in the city of origin has ex . 50 pokestops , and the city has 10km² takes a 2-minute average for each one , and if you keep running them direct , perhaps the range is considaderado to crash in the collection system , so ip is out, and also the duplication of code ip.

**Comment by AlexCtasy** [comment]
I agree with that, cause that i prefer gps in my country always in the same zone of cords. No way to get detected and the gps u can save the zone for option forever so, u will be logged there forever like u lives in that place, is wonderful :D

**Comment by saintv** [comment]
i'm in shinjuku park right now and only 42k exp/h with 60 kh/h...i rather go back to pier 39

**Comment by Neshira** [comment]
If you're in the US you can use Las Vegas strip or Central park. Both of them are good locations and as long as you try to wait a few hours between botting and logging on you can use the "i was on vacation" excuse if Niantic decides to pursue. Never go overseas or to a different country, try to stay in your own state if possible. At least for me it's not, I'm in Tennessee so there aren't many stops here.
